.github/workflows/style.yml

```diff
@@ -0,0 +1,62 @@
+name: style
+
+# Controls when the action will run. 
+on:
+  push:
+    branches: [ master, dev ]
```

Comment:
let's add `main` here too. Some repos might use main instead of master in the future.